### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by chvalean**
This is taken from the WALA agent repo, we can benefit to have a uniform PR process.
https://github.com/Azure/WALinuxAgent

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+### PR information
+- [ ] The title of the PR is clear and informative.
+- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
+- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
+- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
+- [ ] Verify PR checker results and fix any coding errors.
+- [ ] Included LISAv2 sample test results to demonstrate code functionality. This must be done on at least CentOS/RedHat and Ubuntu.
+
+### Quality of Code and Contribution Guidelines
+- [ ] I have read the [contribution guidelines](https://github.com/Azure/WALinuxAgent/blob/master/.github/CONTRIBUTING.md).
```
[the remark this answers is not in the document]
Comment (chvalean) [reply .github/PULL_REQUEST_TEMPLATE.md]: will have to take that md and make minor changes to it, thanks!

**Comment by chvalean** [comment]
@juhlee-microsoft can you please proofread this and sign-off?
New PRs will be presented with this template.

**Comment by juhlee-microsoft** [comment]
> @juhlee-microsoft can you please proofread this and sign-off?
> New PRs will be presented with this template.

OK, let's give a try. @chvalean

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+### Pull Requests
+
+You can find all of the pull requests that have been opened in the [Pull Request](https://github.com/LIS/LISAv2/pulls) section of the repository.
+
+To open your own pull request, click [here](https://github.com/LIS/LISAv2/compare). When creating a pull request, keep the following in mind:
+- Make sure you are pointing to the fork and branch that your changes were made in
+- Choose the correct branch you want your pull request to be merged into
+- The pull request template that is provided **should be filled out**; this is not something that should just be deleted or ignored when the pull request is created
+    - Deleting or ignoring this template will elongate the time it takes for your pull request to be reviewed
+<!-- - The SLA for reviewing pull requests is **two business days** -->
```
Comment (juhlee-microsoft) [inline .github/CONTRIBUTING.md]: small change - 2 business days
big changes - 4  ~ 5 business days

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Please add an informative description that covers that changes made by the pull request.
+This checklist is used to make sure that common issues in a pull request are addressed.
+This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.
+-->
+
+---
+
+### PR information
+- [ ] The title of the PR is clear and informative.
+- [ ] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
```
[the remark this answers is not in the document]
Comment (chvalean) [reply .github/PULL_REQUEST_TEMPLATE.md]: this is a generic document on another repo, wala repo also linked it from the original source.

**Comment by chvalean** [comment]
@juhlee-microsoft @iamshital please check and approve, all comments have been addressed.

**Pull request merged**